Components/input/input.css

```diff
@@ -0,0 +1,71 @@
+.input-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1.5rem;
+    margin-bottom: 1.8rem;
+    flex-wrap: wrap;
+    box-shadow: 1px 1px 2px var(--shadow-color);
+    border-radius: 10px;
+    border: 2px solid var(--light-border-color);
+    background-color: var(--background-color);
+    height: auto;
+    width: 100%;
+}
+
+.input-section{
+    font-size: .8rem;
+    line-height: 1.25rem;
+    margin: 0 1.25rem 0.625rem;
+    width: 100%;
+}
+
+.input-title{
+    display: block;
+    margin-bottom: .5rem;
+}
+
+.text-input{
+    border: 1px solid var(--shadow-color);
+    padding: .6rem .7rem;
+    float: left;
+    font-size: .8rem;
+    margin: 0.313rem 0;
+    width: 100%;
+    transition: .3s all ease;
+    outline: none;
+    border-radius: 2px;
+}
+
+.flex-container{
+    display: flex;
+    /* flex-direction: column; */
+}
+
+.input-section-invalid{
+    color: var(--danger-color);
+}
+.invalid-input{
+    border: 1px solid var(--danger-color);
+}
+
+.invalid-msg{
+    margin-top: .5rem;
+    color: var(--danger-color);
+}
+
+.radio-title, .checkbox-title{
+    display: flex;
+    align-items: center;
+    margin: 10px;
+    font-size: 13px;
+}
+
+.radio-section input,.checkbox-section input{
+    cursor: pointer;
+    margin: 0 10px;
+}
+
+.disable{
+    color: var(--disable-button-color);
+}
```

Comment:
add new line at the end of the file
you can also enable addFinalNewLine in vscode settings